src/AspNet.Security.OAuth.HubSpot/HubSpotAuthenticationOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using static AspNet.Security.OAuth.HubSpot.HubSpotAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.HubSpot;
+
+/// <summary>
+/// Defines a set of options used by <see cref="HubSpotAuthenticationHandler"/>.
+/// </summary>
+public class HubSpotAuthenticationOptions : OAuthOptions
+{
+    public HubSpotAuthenticationOptions()
+    {
+        ClaimsIssuer = HubSpotAuthenticationDefaults.Issuer;
+        AuthorizationEndpoint = HubSpotAuthenticationDefaults.AuthorizationEndpoint;
+        TokenEndpoint = HubSpotAuthenticationDefaults.TokenEndpoint;
+        CallbackPath = HubSpotAuthenticationDefaults.CallbackPath;
+        UserInformationEndpoint = HubSpotAuthenticationDefaults.UserInformationEndpointFormat;
+        ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "user");
```

Comment:
Just a small change for readability/consistency.

```suggestion
        ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, "user");

```